orderbook/src/main.rs

```diff
@@ -8,6 +8,6 @@ use std::sync::Arc;
 async fn main() {
     let orderbook = Arc::new(OrderBook::default());
     let filter = api::handle_all_routes(orderbook);
-    let result = warp::serve(filter).bind(([127, 0, 0, 1], 8080)).await;
+    let result = warp::serve(filter).bind(([0, 0, 0, 0], 8080)).await;
```

Comment:
This was needed since loopback (127.0.0.1) is not accessible from the host system (your browser). This is also the IP at which we bind the price estimator.